src/fuzzyfile.js

```diff
@@ -56,16 +66,21 @@ export default {
       "str": "dbuggen tidning",
       "color": "#0ab87a",
       "href": "https://dbu.gg"
-    },{
-      "name": "Motioner",
-      "str": "motioner",
-      "color": "white",
-      "href": "https://motioner.datasektionen.se"
     },{
       "name": "Metaspexet",
       "str": "metaspexet",
       "color": "red",
       "href": "http://metaspexet.se"
+    },{
+      "name": "Tiki - Biljettsystem",
+      "str": "biljetter tiki biljettsystem",
+      "color": "#000000",
+      "href": "https://tiki.datasektionen.se/"
+    },{
+      "name": "MatterMost",
```

Comment:
```suggestion
      "name": "Mattermost",
```
Niet cyka blyat pizdets rush b yebat